packages/image/src/constants/fitModes.js

```diff
@@ -1,6 +1,7 @@
 const fitModes = {
-  COVER: "cover",
   CONTAIN: "contain",
+  FILL: "fill",
+  COVER: "cover",
```

Comment:
As commented in the Readme, there's another possible value: `scale-down`.
Please add it here and include a small image to the stories.